### PR TITLE
Image viewer tests

### PR DESCRIPTION
**Pull request by dave3d**

### CODE REVIEW

**Comment by blowekamp** [comment]
ghost flow is detecting tracing white space. We should likely follow PEP8 in the tests and run the Python linter on them too.

File: Testing/Unit/Python/dummy_viewer.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import sys
+import os
+
+
+out = ""
+
+if len(sys.argv) > 1:
+    for x in sys.argv[1:]:
+        out = out + ' ' + x
+
```
Comment (zivy) [inline Testing/Unit/Python/dummy_viewer.py:12]: Instead of the loop:
" ".join(sys.argv[1:])

File: Testing/Unit/Python/dummy_viewer.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+print(out)
+os.environ['DUMMY_SHOW'] = x
```
Comment (zivy) [inline Testing/Unit/Python/dummy_viewer.py:15]: The variable 'x' doesn't exist till assigned a value in the loop. If no argument is given this causes an exception. Was the intent to use the 'out' variable and not the 'x' variable?

File: Testing/Unit/Python/sitkExternalViewerTest.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    """Test external viewer launch"""
+
+    IMG = None
+
+    @classmethod
+    def setUpClass(cls):
+        ExternalViewerTest.IMG = sitk.GaussianSource()
+        this_dir = os.path.dirname(os.path.abspath(__file__))
+
+        cmd = 'python ' + this_dir + '/dummy_viewer.py'
```
Comment (zivy) [inline Testing/Unit/Python/sitkExternalViewerTest.py:37]: cmd = ' python ' + os.path.join(this_dir, 'dummy_viewer.py')

File: Testing/Unit/Python/dummy_viewer.py (hunk 1 above)
Comment (dave3d) [reply Testing/Unit/Python/dummy_viewer.py:12]: Clearly my brain doesn't work in a Pythonic manner.

**Comment by dave3d** [comment]
> ghost flow is detecting tracing white space. We should likely follow PEP8 in the tests and run the Python linter on them too.

Where is it?  I ran flake8 on my stuff before I checked it in and didn't see anything.

File: Testing/Unit/Python/dummy_viewer.py (hunk 1 above)
Comment (zivy) [reply Testing/Unit/Python/dummy_viewer.py:12]: No worries. I guess you're writing C++ in Python. My original R code looked like Python in R till Richard showed me the R way of doing things. Each language and its unique quirks.

**Pull request merged**